content/intro-to-storybook/vue/en/simple-component.md

```diff
@@ -79,7 +79,7 @@ export const Default = () => ({
   template: taskTemplate,
```

Comment:
@sshiling if you could add the file name that is being changed as a comment it would be awesome. I think that it's a nice touch for the reader to have a point of reference. For a better understanding of what i'm saying if you take a look at the [spanish translation](https://www.learnstorybook.com/intro-to-storybook/vue/es/simple-component/) you'll see what i mean. Each snippet starts with the file name that is being modified/created. Do you mind applying these changes in this pr? And also if you're willing apply it to rest the english version, so that it all starts aligning itself? If you don't have time, i can make the changes later, that's no problem

Reply:
Good idea with comments at the start of the code snippets. I'll do it tomorrow during the day.

Reply:
@jonniebigodes comments done.